### ISSUE TITLE
Fix detection whether need the accessible name on SVG elements

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#592]
- Markuplint version: `3.0.0-rc.0`
- Parser lang: HTML
- Node.js version: `16`
- OS: macOS

## Describe the bug

Warned on [`require-accessible-name`](https://markuplint.dev/rules/require-accessible-name)

## Code Example

```html
<a href="path/to">
  <svg aria-label="i-have-a-name">
    <path ...>
  </svg>
</s>
```

## The current behavior

It said `<path>` requires the accessible name.

## The expected behavior

`<path>`'s role is `graphics-symbol`. So I guess it doesn't need an accessible name.

I must research SVG's ARIA roles.
